### PR TITLE
Serpent Games: Switch to Binary Targets

### PR DESCRIPTION
**Pull request by nbrochu**
Switched my 4 merged games to use `BinaryTarget`. All tested to work fine post-change.

### CODE REVIEW

File: HintMachine/Games/DorfromantikConnector.cs [hunk 1]
```diff
@@ -45,7 +53,9 @@ public DorfromantikConnector()
 
         public override bool Connect()
         {
-            _ram = new ProcessRamWatcher("Dorfromantik", "mono-2.0-bdwgc.dll");
+            _ram = new ProcessRamWatcher();
+            _ram.SupportedTargets.Add(GAME_VERSION_STEAM);
```
Comment (Dinopony) [inline HintMachine/Games/DorfromantikConnector.cs]: There is a simpler syntax to accomplish this: 
```cs
_ram = new ProcessRamWatcher(GAME_VERSION_STEAM);
```

File: HintMachine/Games/IslandersConnector.cs [hunk 2]
```diff
@@ -43,7 +51,9 @@ public IslandersConnector()
 
         public override bool Connect()
         {
-            _ram = new ProcessRamWatcher("ISLANDERS", "mono-2.0-bdwgc.dll");
+            _ram = new ProcessRamWatcher();
+            _ram.SupportedTargets.Add(GAME_VERSION_STEAM);
```
Comment (Dinopony) [inline HintMachine/Games/IslandersConnector.cs]: Same comment as in Dorfromantik

File: HintMachine/Games/LuckBeALandlordConnector.cs [hunk 3]
```diff
@@ -27,7 +34,9 @@ public LuckBeALandlordConnector()
 
         public override bool Connect()
         {
-            _ram = new ProcessRamWatcher("Luck be a Landlord");
+            _ram = new ProcessRamWatcher();
+            _ram.SupportedTargets.Add(GAME_VERSION_STEAM);
```
Comment (Dinopony) [inline HintMachine/Games/LuckBeALandlordConnector.cs]: Same comment as in Dorfromantik

File: HintMachine/Games/PacManChampionshipEditionDXConnector.cs [hunk 4]
```diff
@@ -24,7 +31,9 @@ public PacManChampionshipEditionDXConnector()
 
         public override bool Connect()
         {
-            _ram = new ProcessRamWatcher("PAC-MAN");
+            _ram = new ProcessRamWatcher();
+            _ram.SupportedTargets.Add(GAME_VERSION_STEAM);
```
Comment (Dinopony) [inline HintMachine/Games/PacManChampionshipEditionDXConnector.cs]: Same comment as in Dorfromantik

**Comment by nbrochu** [comment]
I actually prefer doing it the longer, more explicit way because if another PC platform is added later, nothing need to be reworked, you only add. I don't like this inline stuff at all and don't think it adds any value personally.

**Comment by nbrochu** [comment]
I misunderstood what shorthand meant here but I still don't think it adds much value if you have to revert to explicitly adding targets anyway when a second one is added.

**Comment by nbrochu** [comment]
It's now using the shorthand

**Pull request merged**